proto/lighthouse-result.proto

```diff
@@ -42,6 +42,10 @@ enum LighthouseError {
   PARSING_PROBLEM = 15;
   // The trace data failed to stream over the protocol.
   READ_FAILED = 16;
+  // Used when security error prevents page load.
+  INSECURE_DOCUMENT_REQUEST = 17;
```

Comment:
is this spillover from the other PR? :)

Reply:
Well, it was missed in #6300. nbd since LR's LH bundle hasn't been updated since.